### DISCUSSION TITLE
Is this possible to add to SD.NXT?

### DISCUSSION THREAD

**Discussion by Nacurutu** [vladmandic/sdnext#2524] [Q&A] [answered]
I don't know how this works and maybe I'm being dumb asking this but, is there any chance to have this?

https://www.audiocipher.com/post/suno-ai-chirp

https://app.suno.ai/

**Comment by vladmandic** [comment]
i don't know what suno.ai is an i'm not about to sign-up for it just to see what it is.

**Reply by Nacurutu** [reply]
oh, sorry..

Text to speech and Text to music.

Bark and Chirp

https://www.audiocipher.com/post/suno-ai-chirp

You can also generate from their Discord server. I'm asking if it is possible to have it locally on SD.NXT

**Reply by vladmandic** [reply, marked as answer]
there are no plans to extend sdnext to text-to-audio, that would be a completely different use-case.
